go.mod

```diff
@@ -3,10 +3,17 @@ module github.com/openshift/client-go
 go 1.16
 
 require (
-	github.com/openshift/api v0.0.0-20210831091943-07e756545ac1
+	github.com/go-logr/logr v1.1.0 // indirect
+	github.com/google/go-cmp v0.5.6 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/openshift/api v0.0.0-20210913135842-b9ab6ddbf49c
```

Comment:
This would be the only expected change here, nothing else should show up. What did you use to update these?
The recommended approach is to use the following three commands:
```
go mod edit -require github.com/openshift/api@master
go mod tidy 
go mod vendor
```

Reply:
I ran the following. `go get -u github.com/openshift/api@master`

I'll resubmit after removing the `-u`. Then `go mod vendor` and `go mod verify`.